automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/PageUtil.scala

```diff
@@ -1,13 +1,13 @@
 package org.broadinstitute.dsde.firecloud.page
 
+import org.broadinstitute.dsde.firecloud.FireCloudView
 import org.openqa.selenium.WebDriver
 import org.scalatest.selenium.Page
-import org.scalatest.selenium.WebBrowser.go
 
 /**
   * Mix-in utilities for ScalaTest's Page.
   */
-trait PageUtil[P <: Page] { self: P =>
+trait PageUtil[P <: Page] extends FireCloudView { self: P =>
```

Comment:
`PageUtil` was intended to be not FireCloud-specific so I’d rather it not extend `FireCloudView`. I like the standard `awaitReady` but I’d like to make some tweaks to the code structure, maybe even just renaming things, to make a clean separation between FireCloud support and more general extensions to the Selenium API.